tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -2453,6 +2453,45 @@ describe('runs_table', () => {
           );
         });
       });
+
+      it('does not sort because you click on the filter menu button', () => {
+        store.overrideSelector(
+          getRunHparamFilterMap,
+          buildHparamFilterMap([
+            [
+              'foo',
+              buildDiscreteFilter({
+                possibleValues: ['faz', 'bar', 'baz'],
+              }),
+            ],
+          ])
+        );
+        store.overrideSelector(
+          getRunMetricFilterMap,
+          buildMetricFilterMap([
+            [
+              'acc',
+              buildIntervalFilter({
+                includeUndefined: false,
+                filterLowerValue: 0.25,
+                filterUpperValue: 1,
+              }),
+            ],
+          ])
+        );
+        const fixture = createComponent(TEST_HPARAM_SPECS, TEST_METRIC_SPECS);
+        fixture.detectChanges();
+
+        const columnHeaders = fixture.nativeElement.querySelectorAll(
+          Selector.HEADER_COLUMN
+        );
+        columnHeaders[3].querySelector('button').click();
+        columnHeaders[4].querySelector('button').click();
```

Comment:
It looks like there are only 4 elements with [role="columnheader"] 
Did I miss anything?

Reply:
0th: checkbox
1st: name
2nd: something I am forgetting (time?)
3rd: hparam
4th: metric.

(since I added the `getRunMetricFilterMap`).